templates/libvirt/xen.xml

```diff
@@ -179,13 +184,13 @@
                     {% endif %}
                     {% if vm.netvm %}
```

Comment:
This all is added only if netvm is set. You need to add a case for no-netvm too.